src/components/SingleDatePicker.jsx

```diff
@@ -203,7 +202,7 @@ class SingleDatePicker extends React.PureComponent {
     });
 
     onFocusChange({ focused: false });
-    onClose({ startDate, endDate });
+    onClose({ date});
```

Comment:
```suggestion
    onClose({ date });
```

Reply:
done

Reply:
https://github.com/airbnb/react-dates/blob/11de2016bf6e7111ddebd5a7f7fc1fe929a3600c/test/components/SingleDatePicker_spec.jsx#L407
in this test suite should i add a test for `calls props.onFocusChange ` with  `date` props ?

Reply:
that sounds great, yes

Reply:
can i run particular test file?

Reply:
done